From e8bef5f863461fecfbcfff61661f111aa514ad8b Mon Sep 17 00:00:00 2001
From: Arnon Kaufman <akaufman@habana.ai>
Date: Mon, 08 Jul 2024 10:46:52 +0300
Subject: [PATCH] [SW-0] scripts: single arch cleanup

After removing Goya and Greco there is no need to support single arch

Change-Id: Ifa7366c473eb3572432c2dcf9873a861973283de
Signed-off-by: Arnon Kaufman <akaufman@habana.ai>
---

diff --git a/scripts/make/hl-common/Makefile b/scripts/make/hl-common/Makefile
index 5e0dcaa..6124aa4 100644
--- a/scripts/make/hl-common/Makefile
+++ b/scripts/make/hl-common/Makefile
@@ -1,6 +1,5 @@
 HL_MULTI_ARCH_RELEASE := gaudi2 gaudi3 fs1
 HL_DEFAULT_DEV_ARCH := gaudi2_arm gaudi3_arc fs1_arc
-HL_SINGLE_ARCH_RELEASE :=
 HL_SECURITY_RELEASE :=
 
 RELEASE_JOB_FAKE_SEC := "fake_security=true"
@@ -11,7 +10,7 @@
 RELEASE_JOB_FS_VARS_SEC := $(RELEASE_JOB_FAKE_SEC)&$(RELEASE_JOB_RUN_TEST)&$(RELEASE_JOB_RUN_TEST_BIN)&$(RELEASE_JOB_RUN_SYSTEM_TEST)
 RELEASE_JOB_FS_VARS_SEC := $(RELEASE_JOB_FS_VARS_SEC)&$(RELEASE_JOB_INSTALL_FALSE)&$(RELEASE_JOB_UPDATE_MAP_FALSE)
 
-HL_TARGET_RELEASE := $(HL_SINGLE_ARCH_RELEASE) $(HL_MULTI_ARCH_RELEASE)
+HL_TARGET_RELEASE := $(HL_MULTI_ARCH_RELEASE)
 
 .PHONY: %-release-common
 %-release-common:
@@ -35,8 +34,7 @@
 
 HL_FAKE_SEC ?=
 export STACK_REDUCTION_SIZE ?= 0
-HL_TARGET_ARCH := $(addsuffix %,$(HL_SINGLE_ARCH_RELEASE)) \
-		  $(addsuffix _%,$(HL_MULTI_ARCH_RELEASE))
+HL_TARGET_ARCH := $(addsuffix _%,$(HL_MULTI_ARCH_RELEASE))
 
 
 define add-hl-prefix
@@ -93,8 +91,7 @@
 		$@-vars_file)
 
 HL_MULTI_ARCH := $(addsuffix -%,$(HL_MULTI_ARCH_RELEASE))
-HL_TARGET_ARCH_PREPARE := $(addsuffix -%,$(HL_SINGLE_ARCH_RELEASE)) \
-			  $(addsuffix _%,$(HL_MULTI_ARCH_RELEASE))
+HL_TARGET_ARCH_PREPARE := $(addsuffix _%,$(HL_MULTI_ARCH_RELEASE))
 
 .PHONY: %-prepare
 TARGET_DEFCONFIG ?= $(strip $(call hl-device, $@))_defconfig
@@ -126,8 +123,7 @@
 	@$(if $(filter $(HL_MULTI_ARCH), $@), \
 		$(MAKE) $(call hl-device, $@)-post-image)
 
-HL_TARGET_ARCH_CLEAN := $(addsuffix -%,$(HL_SINGLE_ARCH_RELEASE)) \
-			$(addsuffix _%,$(HL_MULTI_ARCH_RELEASE))
+HL_TARGET_ARCH_CLEAN := $(addsuffix _%,$(HL_MULTI_ARCH_RELEASE))
 
 .PHONY: %-distclean
 %-distclean:
